test/extended/image_ecosystem/sample_repos.go

```diff
@@ -183,75 +183,6 @@ var _ = g.Describe("[image_ecosystem][Slow] openshift sample application reposit
-		sampleRepoConfig{
-			repoName:               "dancer-mysql",
-			templateURL:            "https://raw.githubusercontent.com/openshift/dancer-ex/master/openshift/templates/dancer-mysql.json",
-			buildConfigName:        "dancer-mysql-example",
-			serviceName:            "dancer-mysql-example",
-			deploymentConfigName:   "dancer-mysql-example",
-			expectedString:         htmlCountValueNonZeroRegexp,
-			appPath:                "",
-			dbDeploymentConfigName: "database",
-			dbServiceName:          "database",
-		},
-	))
-
-	// test the no-db templates too
-	g.Describe("[image_ecosystem][python] test python images with django-ex repo", NewSampleRepoTest(
-		sampleRepoConfig{
-			repoName:               "django",
-			templateURL:            "https://raw.githubusercontent.com/openshift/django-ex/master/openshift/templates/django.json",
-			buildConfigName:        "django-example",
-			serviceName:            "django-example",
-			deploymentConfigName:   "django-example",
-			expectedString:         "Welcome",
-			appPath:                "",
-			dbDeploymentConfigName: "",
-			dbServiceName:          "",
-		},
-	))
-
-	g.Describe("[image_ecosystem][nodejs] images with nodejs-ex repo", NewSampleRepoTest(
-		sampleRepoConfig{
-			repoName:               "nodejs",
-			templateURL:            "https://raw.githubusercontent.com/openshift/nodejs-ex/master/openshift/templates/nodejs.json",
-			buildConfigName:        "nodejs-example",
-			serviceName:            "nodejs-example",
-			deploymentConfigName:   "nodejs-example",
-			expectedString:         "Welcome",
-			appPath:                "",
-			dbDeploymentConfigName: "",
-			dbServiceName:          "",
-		},
-	))
-
-	var _ = g.Describe("[image_ecosystem][php] test php images with cakephp-ex repo", NewSampleRepoTest(
-		sampleRepoConfig{
-			repoName:               "cakephp",
-			templateURL:            "https://raw.githubusercontent.com/openshift/cakephp-ex/master/openshift/templates/cakephp.json",
-			buildConfigName:        "cakephp-example",
-			serviceName:            "cakephp-example",
-			deploymentConfigName:   "cakephp-example",
-			expectedString:         "Welcome",
-			appPath:                "",
-			dbDeploymentConfigName: "",
-			dbServiceName:          "",
-		},
-	))
-
-	var _ = g.Describe("[image_ecosystem][perl] test perl images with dancer-ex repo", NewSampleRepoTest(
-		sampleRepoConfig{
-			repoName:               "dancer",
-			templateURL:            "https://raw.githubusercontent.com/openshift/dancer-ex/master/openshift/templates/dancer.json",
-			buildConfigName:        "dancer-example",
-			serviceName:            "dancer-example",
-			deploymentConfigName:   "dancer-example",
-			expectedString:         "Welcome",
-			appPath:                "",
-			dbDeploymentConfigName: "",
-			dbServiceName:          "",
-		},
-	))
+	// dependency download with perl is intermittently slow enough to blow away the e2e timeouts so we've removed those
```

Comment:
ok......but why drop the other ones? (nodejs, php)

Reply:
ah ... dancer and django are commented out in 4.x ... see https://github.com/openshift/origin/blob/master/test/extended/image_ecosystem/sample_repos.go#L179-L206  ... so I just deleted those here, cause even if we ever add those back in, we'll start by adding in 4.x first and then might backport to 3.11

that said, the ordering in that file is different than nodejs and I inadvertently deleted that one ... it does exist in 4.x ... see https://github.com/openshift/origin/blob/master/test/extended/image_ecosystem/sample_repos.go#L150-L162

so yeah I need to add nodejs back ... php is still there ... still delete perl/python

thanks @bparees

Reply:
oh sorry ... one more detail ... in 4.x we only did the templates that were combos with DBs, but removed the ones without DBs

I was trying to pattern 3.11 like that, like what we did with 4.x

are we OK with that @bparees @adambkaplan ?

Reply:
i.e. only these:  https://github.com/openshift/origin/blob/master/test/extended/image_ecosystem/sample_repos.go#L120-L176

and if I did that, I need to see if we can replace nodejs-mongodb with nodejs-postresql in 3.11 (given the mongo licensing situation)

Reply:
yeah only nodejs-mongo in 3.11 https://github.com/openshift/openshift-ansible/tree/release-3.11/roles/openshift_examples/files/examples/x86_64/quickstart-templates

Reply:
matching 4.x sounds fine to me

Reply:
great ... so other than the nodejs-mongodb vs. nodejs-postrgesql piece I think that is where we are 

first blush I'm not inclined to remove nodejs-mongodb template and add nodejs-postgresql template ... feels more than an e2e stabilization item

you good with that too @bparees ?

Reply:
yup that's fine, thanks for the sanity check